VideoLocker/src/main/java/org/edx/mobile/util/UnicodeChars.java

```diff
@@ -0,0 +1,11 @@
+package org.edx.mobile.util;
+
+
+public class UnicodeChars {
```

Comment:
Suggested javadoc: A collection of Unicode characters that may be ambiguous when rendered directly as literals.